e2e-tests/secure_supply_chain_tests.bats

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bats
 
-FULCIO_AND_REKORD_DATA_DIR=$(readlink -f ~/.config/kubewarden/fulcio_and_rekor_data)
-
 setup() {
     rm -rf ~/.cache/kubewarden
-    rm -rf ${FULCIO_AND_REKORD_DATA_DIR}
+    rm -rf ~/.config/kubewarden
```

Comment:
Cannot we mess up with our env using the home directory? Using a variable (even if the default value is the home directory), we can make the test portable to run in multiple directory each one with different configs. Maybe I'm being too pedantic here. But I'm just sharing a thought. ;-)

Reply:
That's a good point, done in https://github.com/kubewarden/kwctl/pull/183/commits/13b95ae7fad71e232ad3460a152fdd1ffb743a5d. 
We could depend on bats-file and their [temp_file](https://github.com/ztombol/bats-file#working-with-temporary-directories), but I don't think it's worth the effort.